src/app/lib/access/forced_update/current_version_repository.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:app/access/forced_update/data/version.dart';
+import 'package:flutter/foundation.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+/// Service to get what is the current version of the app.
```

Comment:
The word service here makes me wonder if it's appropriate since it's a repository and from my understanding repos are used by services

Reply:
Yeah, I think it was originally a service but then I decided to make it a repository and changed the name. But apparently forgot to change the comment on top.